src/StatusQ/Popups/statusModal/StatusModalHeader.qml

```diff
@@ -36,6 +38,18 @@ Rectangle {
         }
     }
 
+    Rectangle {
+        anchors.bottom: parent.bottom
+        width: parent.width
+        height: parent.radius
+        color: parent.color
+
+        StatusModalDivider {
+            anchors.bottom: parent.bottom
+            width: parent.width
+        }
+    }
```

Comment:
Any reason this was moved?

Reply:
bc of `z` order this line hide part of header.